rpm/PackageKit.spec

```diff
@@ -193,7 +156,6 @@ update-mime-database %{_datadir}/mime &> /dev/null || :
 %dir %{_sysconfdir}/PackageKit
 %dir %{_localstatedir}/lib/PackageKit
 %dir %{_localstatedir}/cache/PackageKit
-%dir %{_localstatedir}/cache/PackageKit/downloads
 %dir %{_libdir}/packagekit-backend
 %config %{_sysconfdir}/PackageKit/*.conf
 %config %{_sysconfdir}/dbus-1/system.d/*
```

Comment:
Dbus services should go to `%{_datadir}` as done in upstream, Fedora and already done for Polkit.

Reply:
Upstream defaults to datadir too: https://github.com/PackageKit/PackageKit/blob/b54014f6e7650ed28f16714c95fd7f84d65bc613/data/meson.build#L25.

Reply:
And upstream defaults sysconfdir: https://github.com/PackageKit/PackageKit/blob/b54014f6e7650ed28f16714c95fd7f84d65bc613/data/meson.build#L12

As I said, it uses both.

Reply:
> And upstream defaults sysconfdir: https://github.com/PackageKit/PackageKit/blob/b54014f6e7650ed28f16714c95fd7f84d65bc613/data/meson.build#L12

that's for the config, I'm talking about the dbus sevice.

Reply:
Sounds like you didn't test how this really ends up packaged.

```
$ rpm -qpil RPMS/PackageKit-1.2.5-1.aarch64.rpm |grep dbus-1
/etc/dbus-1/system.d/org.freedesktop.PackageKit.conf
/usr/share/dbus-1/interfaces/org.freedesktop.PackageKit.Transaction.xml
/usr/share/dbus-1/interfaces/org.freedesktop.PackageKit.xml
/usr/share/dbus-1/system-services/org.freedesktop.PackageKit.service
```